website/src/views/conceptMap.js

```diff
@@ -34,8 +32,9 @@ export default function ConceptMap() {
         let mounted = true;
         if (mounted) {
             setLoading(true);
-            apiv2.get('/students/' + selectedStudent + '/progressquerystring').then((res) => {
-                setStudentMastery(res.data)
+            apiv2.get(`/students/${selectedStudent}/progressquerystring`).then((res) => {
+                //.substring(0, 6) for quest data only
+                setStudentMastery(res.data); //remove substring when we are ready for midterm and postterm
```

Comment:
Is there a reason why we are doing this and not just including it from the get go?  Also, if this is a task to be done later, it should be on the project board not as a comment (we won't see it / remember it if its a comment)

Reply:
So what do you want to include on the launched version? All the exams or only a specific exam?

Reply:
All data currently on the sheet

Reply:
In that case, I can delete the comments because the application already does this.